docs/concepts.rst

```diff
@@ -539,13 +539,22 @@ Most Python objects can be serialized with Python's built-in `pickle
 <https://docs.python.org/3/library/pickle.html>`_ module.  However, for some
 object types it's more efficient or more idiomatic to use a different format.
 There are also some types of objects that can't be pickled at all.  Bionic uses
-``pickle`` by default, but handles some types specially: `Pandas
-<https://pandas.pydata.org/>`_ DataFrames are serialized in the `Parquet
-<https://parquet.apache.org/>`_ format, while `Pillow
-<https://pillow.readthedocs.io/en/stable/>`_ Images are serialized as `PNG
-<https://en.wikipedia.org/wiki/Portable_Network_Graphics>`_\ s.  You can
-explictly specify a serialization strategy for an entity by attaching a
-`Protocol`_ to its definition.
+``pickle`` by default, but handles some types specially:
+
+- `Dask <https://docs.dask.org/>`_ Dataframes are serialized in the `Parquet
+  <https://parquet.apache.org/>`_ format,
+- `GeoPandas <https://geopandas.org/>`_ Dataframes are serialized as `SHP
+  <https://en.wikipedia.org/wiki/Shapefile#Shapefile_shape_format_(.shp)>`_\ s,
+- `NumPy <https://numpy.org/>`_ Arrays are serialized in the `NPY
+  <https://numpy.org/devdocs/reference/generated/numpy.lib.format.html#npy-format>`_
+  format,
+- `Pandas <https://pandas.pydata.org/>`_ DataFrames are serialized in the
+  `Parquet <https://parquet.apache.org/>`_ format,
+- `Pillow <https://pillow.readthedocs.io/en/stable/>`_ Images are serialized as
```

Comment:
We might as well use the same sentence structure for each of these bullet points. Maybe "$LIBRARY $OBJECTs are serialized using $FORMAT."?

(Also, let's put a period at the end of each bullet point.)